### ISSUE TITLE
[new method] window._shareddata

### ISSUE THREAD

**Issue by becauseimnoob** [InstaPy/InstaPy#6700]
It will be a solution to your problems, but I don't know how to apply the code. Just add `?__a=1&__d=1` at the end of the url

example https://www.instagram.com/rterdogan/?__a=1&__d=1

![resim](https://user-images.githubusercontent.com/116311873/229247392-b5a5d7e7-c6fc-4fca-81f0-a32881e26a88.png)

#6647  #6615 

@elulcao  @timgrossmann

**Comment by official-antique** [comment]
@becauseimnoob the data in the screenshot isn't `window._sharedData` but a JSON representation of your profile (can also be appended to any Instagram endpoint iirc).

You can find `window._sharedData` by tapping F12 and going to the Console tab, from there enter `console.log(window._sharedData)` and hit Enter. The shared data is there but the important part; `entry_data` is not.

**Comment by becauseimnoob** [comment]
You can also obtain data by using this information. This way you don't have to rewrite the code. https://github.com/Krisseck/InstaPy/blob/master/instapy/util.py#L2657 @Krisseck seems to have fixed it but there are a few places to fix it.

**Comment by becauseimnoob** [comment]
> @becauseimnoob the data in the screenshot isn't `window._sharedData` but a JSON representation of your profile (can also be appended to any Instagram endpoint iirc).
> 
> You can find `window._sharedData` by tapping F12 and going to the Console tab, from there enter `console.log(window._sharedData)` and hit Enter. The shared data is there but the important part; `entry_data` is not.

You can also obtain data by using this information. This way you don't have to rewrite the code. https://github.com/Krisseck/InstaPy/blob/master/instapy/util.py#L2657 @Krisseck seems to have fixed it but there are a few places to fix it.

**Comment by official-antique** [comment]
Yes that was also in my original fix prior to me deciding to make my own project using Instagrapi.

That's a long way from making this project work though, sadly.

**Comment by becauseimnoob** [comment]
I have tried and used all existing projects. All simple and far from instapy. This problem can be overcome. There is no such great software(instapy), here are some of the ones I use

1. https://github.com/GramAddict/bot  (It has basic features. Specific things cannot be done. For example, if you don't want users to be interacted with, you need to change the code.)
2. https://github.com/alexal1/Insomniac (same as above :D Some features are paid.)
3. https://github.com/mifi/instauto (Simple features..)
4. https://github.com/adw0rd/instagrapi (Those who make bots with API did not end well. Also bot with api is like an invitation code for instagram to punch you :) )


In short, there is a great library here, why wait for the development process of projects that have no features? Today, even children who are new to python can create an Instagram like comment bot. But instapy is more than that, please don't be offended by what I'm saying. (I'm not telling you, I'm speaking generally)


As we have overcome all the obstacles that have come our way over the years, we will overcome this problem as a community. But sooner or later

**Comment by elchupacabrinski** [comment]
sad that this repo is dead
